### PR TITLE
completed object manipulation 1

### PR DESCRIPTION
**Pull request by hnguy96**
Screen Shot 2021-03-21 at 11 03 06 PM

### CODE REVIEW

File: object-manipulation-1/src/get-value.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* exported getValue */
+function getValue(object, key) {
+  for (var key2 in object) {
+    if (object[key] === object[key2]) {
+      return object[key];
+    }
+  }
+  return undefined;
```
Comment (codingcodymiller) [inline object-manipulation-1/src/get-value.js:8]: This function shouldn't require a loop. You have a parameter for the object and a parameter for the property name, you should just access that value and return it

File: object-manipulation-1/src/set-value.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* exported setValue */
+function setValue(object, key, value) {
+  object[key] = value;
+  if (Object.keys(object).length !== 0) {
+    return object;
+  }
+  return undefined;
```
Comment (codingcodymiller) [inline object-manipulation-1/src/set-value.js:7]: This can be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my ntoes

File: object-manipulation-1/src/set-value.js (hunk 2 above)
Comment (hnguy96) [reply object-manipulation-1/src/set-value.js:7]: are you talking about the condition in the `if` statement specifically?

**Comment by hnguy96** [comment]
Screen Shot 2021-03-23 at 9 31 51 AM
Screen Shot 2021-03-23 at 9 37 56 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**